### PR TITLE
Support for vertical polygons

### LINKED ISSUE #3349: KML renders incorrectly - vertical polygons
The attached KML does not render correctly because it contains polygons that hare mostly perpendicular to the surface of the earth.

[2_1450116435.kml.txt](https://github.com/AnalyticalGraphicsInc/cesium/files/62883/2_1450116435.kml.txt)

### PR DESCRIPTION
**Pull request by hpinkos**
Fixes #3349

We can now render this crazy KML file:
image

I connected `CoplanarPolygonGeometry` to the Entity API.  
We use it instead of `PolygonGeometry` when `perPositionHeight: true` and `extrudedHeight: undefined`.

TODO
- [x] Merge #6787 
- [x] Update CHANGES

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @hpinkos!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/entity-polygon/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by mramato** [comment]
Update CHANGES and specifically mention that this improves KML compatibility.

**Comment by hpinkos** [comment]
@mramato this is ready now

File: Apps/Sandcastle/gallery/Polygon.html [hunk 1]
```diff
@@ -108,8 +110,22 @@
     }
 });
 
-viewer.zoomTo(viewer.entities);
-//Sandcastle_End
+var cyanPolygon = viewer.entities.add({
+    name : 'Orange polygon with per-position heights and outline',
```
Comment (mramato) [inline Apps/Sandcastle/gallery/Polygon.html]: This name needs to be updated.

**Comment by mramato** [comment]
Can we add a test for this?  The rest of the updater is pretty well covered by tests I believe.

**Comment by mramato** [comment]
Looks great @hpinkos, just those couple of comments.

**Comment by hpinkos** [comment]
@mramato ready

**Comment by mramato** [comment]
Thanks @hpinkos!

**Comment by silky** [comment]
thanks for adding this @hpinkos !

**Pull request merged**